src/controllers/accounts.js

```
@@ -98,6 +99,47 @@ accountsController.editAccount = function(req, res, next) {
     });
 };
 
+accountsController.postEdit = function(req, res, next) {
+    var user = req.user;
+    if (_.isUndefined(user) || !permissions.canThis(user.role, 'account:edit')) {
+        req.flash('message', 'Permission Denied.');
+        return res.redirect('/accounts');
+    }
+
+    var self = this;
```

Biome (1.9.4)

[error] This aliasing of this is unnecessary.

Arrow functions inherits `this` from their enclosing scope.
Safe fix: Use this instead of an alias.


(lint/complexity/noUselessThisAlias)